dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -790,6 +790,12 @@ class StartupTest {
           }
         }
 
+        try {
+          await devices.deviceReady(device.deviceId);
```

Comment:
Rather than adding it to individual tests can we add this check to the method uninstalling the app? Something like

- uninstall app
- sleep for 1 sec
- await devices.deviceReady(device.deviceId);

Reply:
Oh yeah I can do that. That is the better method.

Reply:
But I think the call to uninstall the app is made by calling flutter directly. Not in the Devices.dart class.

Reply:
Isn't this call the one making the device not available https://cs.opensource.google/flutter/flutter/+/master:dev/devicelab/lib/framework/devices.dart;l=838?q=force&ss=flutter%2Fflutter:dev%2Fdevicelab%2F?

Reply:
I believe that stops the application and not the device?

Reply:
I believe from the logs the device becomes unavailable after the force-stop, do you have one of the logs handy?